sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3494,14 +3494,27 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       Seq(Row(Map[Int, Int]()), Row(Map(1 -> 2))))
   }
 
-  test("SPARK-31242: clone SparkSession should respect sessionInitWithConfigDefaults") {
-    // Note, only the conf explicitly set in SparkConf(e.g. in SharedSparkSessionBase) would cause
-    // problem before the fix.
-    withSQLConf(SQLConf.CODEGEN_FALLBACK.key -> "true") {
-      val cloned = spark.cloneSession()
-      SparkSession.setActiveSession(cloned)
-      assert(SQLConf.get.getConf(SQLConf.CODEGEN_FALLBACK) === true)
-    }
+  test("Perform propagating empty relation after RewritePredicateSubquery") {
```

Comment:
can we add a UT in `RewriteSubquerySuite`?

Reply:
move or add?